content/en/service_management/app_builder/build.md

```diff
@@ -76,13 +76,31 @@ UI components can trigger reactions on an [Event][11].
 
 You can use [JavaScript Expressions][13] anywhere in App Builder to create custom interactions between the different parts of your app.
 
-### Interact with an app in JSON
 
-#### Edit an app
+## View app version history
+
+App Builder keeps a record of every saved version of your app.
+
+To view the version history for your app, in the left-hand menu of your app, click the version history icon {{< img src="service_management/app_builder/version-history-icon.png" inline="true">}}.
+
+The UI displays up to 50 saved or published versions of your app, along with the icon of the user who saved or published the version:
+
+{{< img src="service_management/app_builder/version-history-example.png" alt="An example App Builder version history list with two items, the current version and a previous version" style="width:70%;" >}}
+
+You can perform the following operations:
+
+* To view an app version, click the version in the list.
+* To restore an app version, select the version, then click **Restore Version** in the upper-right.
+* To clone an app version, select the version, then click **Clone Version** in the upper-right.
+
+
+## Interact with an app in JSON
+
+### Edit an app
 
 To edit an app with JSON, click the cog (**Settings**) icon and select **Switch to JSON**. The **Switch to GUI** option in the settings menu takes you back to the GUI editor.
 
-#### Copy or back up an app
+### Copy or back up an app
```

Comment:
Out of scope but since it's in the diff: Should this be "Export an app"/"Export app definitions", export aligns better with the terminology in workflows and is more clearly distinct from cloning than "copy" is IMO